tests/featureTesting/main.py

```diff
@@ -3,17 +3,19 @@
 """
 import os
 import run_cli
-import compare
-from config import config
-from helper import helper
+import compare_csv
+from helper import helper as hf
+import constants_feature_testing as cs
+import compare_json
```

Comment:
Please go through this [link](https://stackoverflow.com/questions/20762662/whats-the-correct-way-to-sort-python-import-x-and-from-x-import-y-statement) to see how the import order should be. It should be alphabetically ordered in each category. There are 3 categories, the in-built python packages like `os` etc then there are third party packages like `pandas` etc and then at the end you have in-app module imports. Each category should have a new line separation between them and they should be alphabetically ordered.

Reply:
@harshitcodes 👍 Done